config.go

```diff
@@ -57,5 +59,6 @@ func NewDefaultConfig() Config {
 		EventsPort:      10001,
 		EventsInterface: "eth0",
 		LogOutput:       ioutil.Discard,
+		PollingWaitTime: 500,
```

Comment:
@ch3lo, do you think we could address this point? It's the last piece missing before we can merge.

Thanks!